builds/ops_agent_plugin.sh

```diff
@@ -15,6 +15,6 @@
 
 set -x -e
 DESTDIR=$1
-mkdir -p "$DESTDIR/opt/google-cloud-ops-agent/libexec"
-go build -buildvcs=false -ldflags "-s -w" -o "$DESTDIR/opt/google-cloud-ops-agent/libexec/google_cloud_ops_agent_uap_plugin" \
+mkdir -p "$DESTDIR/opt/google-cloud-ops-agent"
+go build -buildvcs=false -ldflags "-s -w" -o "$DESTDIR/opt/google-cloud-ops-agent/plugin" \
```

Comment:
Can we call it `uap_plugin`, just cause plugin can be used in too many different contexts.

Reply:
Done

Reply:
Actually https://source.corp.google.com/piper///depot/google3/cloud/cluster/guestplatform/demo_plugin/client.go;bpv=1;bpt=1;rcl=720287686;l=129 no, UAP requires that the plugin entrypoint be named 'plugin'.